client/src/main/java/gui/recievers/RandomFleet.java

```diff
@@ -1,12 +1,12 @@
-package gui.generator;
+package gui.recievers;
 
 
-import com.google.gson.Gson;
 import gui.printers.FleetView;
 import gui.printers.ShipView;
+import gui.utility.JSONConverter;
 
 /**
- * GUI-side fleet random generator
+ * GUI-side fleet random recievers
```

Comment:
typo

Reply:
Thanks for spotting. Didn't noticed it. :+1: